e2e/specs/stateless/profileEditor.spec.ts

```diff
@@ -6,9 +6,6 @@ import { emptyAddress } from '@app/utils/constants'
 
 const oldResolver = '0x84eA74d481Ee0A5332c457a4d796187F6Ba67fEB'
 const newResolver = '0x0E801D84Fa97b50751Dbf25036d067dCf18858bF'
-// This not an actual resovler but a dummy address that has been inserted to the second to last known resolver
-// to test the situation where unwrapped do not show a warning when editing profile.
-const dummyRersolver = '0xd7a4F6473f32aC2Af804B3686AE8F1932bC35750'
 
```

Comment:
By removing the dummyResolver and test we can't test that the profile warning does not pop up when name is unwrapped and using second latest resolver.